bat/BLDJVDOC.BAT

```diff
@@ -15,7 +15,7 @@ pushd %~dps0..
 rem generate javadoc
 if not exist ".\docenv\javadoc\" mkdir docenv\javadoc
 cd docenv\javadoc
-call javadoc.exe ..\..\src\*.java || goto error
+call javadoc.exe ..\..\src\*.java -private || goto error
```

Comment:
Add similar change to bash/bldjavadoc. Line 20 there. New line 20 could be
```
javadoc -quiet -Xdoclint:none -d ${DEST} -private src/*.java
```

Reply:
Changed as per your suggestion. Thanks.